src/components/molecules/MonacoPlaceholder/MonacoPlaceholder.tsx

```diff
@@ -16,25 +35,33 @@ export const MonacoPlaceholder = () => {
     <S.Container>
       <div>
         <S.ImageContainer>
-          <S.Image src={MonacoPlaceholderImage} />
+          <S.Image src={MonacoPlaceholderImageNew} />
         </S.ImageContainer>
-        <S.Title>Need inspiration? Try any of these</S.Title>
+        <S.Title>
+          Select a resource on the left to edit
+          <br />
+          Want to go further? Try these shortcuts
+        </S.Title>
+        <S.Text>Everything about your Cluster live</S.Text>
```

Comment:
This one should also be a `InfoLink` and should lead you to the dashboard option from the left menu